cpp/src/cider/exec/nextgen/context/RuntimeContext.cpp

```diff
@@ -119,6 +121,35 @@ Batch* RuntimeContext::getNonGroupByAggOutputBatch() {
         ->extract({buf}, arrow_array->children[i]);
   }
 
+  auto partial_avg_count = std::count_if(
```

Comment:
add some comments here to tell that this is for partial avg result construction.